### DISCUSSION TITLE
UWP page transitions on android?

### DISCUSSION THREAD

**Discussion by PalusiOld** [unoplatform/uno#13044] [Q&A] [answered]
uwp does animate transitions when navigating from one side to the other (something like the page flying in from below, for example). Is there some way to use these animations also in the android app?

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the report. 

The current Fluent `Frame` implementation does not yet provide animations for pages transitions, but on Android, it's possible [enable an native mode](https://platform.uno/docs/articles/guides/native-frame-nav-tutorial.html) that will [show some animations](https://github.com/unoplatform/uno/blob/2dcd6dacad0d35c73fd5bb0f7a2c328625ec0212/src/Uno.UI/NativeFramePresenter.Android.cs#L132).

**Reply by PalusiOld** [reply]
I get the following error when I put the code in the app class:

```
Severity Code Description Project File Line Suppression state
Error CS0234 The type or namespace name 'Window' does not exist in the 'Windows.UI.Xaml' namespace. (An assembly reference may be missing.)
```
